G_channels.md

```diff
@@ -96,17 +95,16 @@ defmodule HelloPhoenix.RoomChannel do
   def join("rooms:lobby", auth_msg, socket) do
     {:ok, socket}
   end
+
```

Comment:
It's common to not have extra blank lines between different clauses of the same function, so I think the original is fine here. (Though I would love to know if I'm wrong about this.)